RELEASE/scripts/sl_ascend.ash

```diff
@@ -12573,34 +12627,65 @@ boolean L11_redZeppelin()
 		return ccAdv($location[A Mob Of Zeppelin Protesters]);
 	}
 
-	if(item_amount($item[lynyrd snare]) > 0 && get_property("_lynyrdSnareUses").to_int() < 3 && my_hp() > 150)
-	{
-		return ccAdvBypass("inv_use.php?pwd=&whichitem=7204&checked=1", $location[Noob Cave]);
-	}
+	// TODO: create lynyrd skin items
```

Comment:
I tried to do this recently, but couldn't without [Armorcraftiness](http://kol.coldfront.net/thekolwiki/index.php?title=Armorcraftiness). Is there another way to create the Lynyrd Skin equipment in Dark Gyfte?

Reply:
That's right, it's not possible in DG but is in other paths.